PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -182,6 +191,10 @@
             // stop listening to `UIView.layoutSubviews` events
             viewLayoutToken = nil
 
+            // stop capturing console logs
+            consoleLogInterceptor?.stopCapturing()
+            consoleLogInterceptor = nil
```

Comment:
do we need to pause if the app is put in the background?

Reply:
i think its a performance decision (is there any impact?), and what type of logs/network requests are happening not being controlled by the app.
is there a bunch of useless system logs etc.

Reply:
yep, can always add it as a config option if people ask for it 👍